contrib/libvirt/files/sysusers.conf

```diff
@@ -0,0 +1,6 @@
+# Create libvirt-qemu system user and libvirt group
+
+u virtqemu - "virtqemu user" /var/lib/libvirt /usr/bin/nologin
```

Comment:
ditto for underscored accounts/groups

Reply:
i'd also call it libvirt

Reply:
default seems to be libvirt-qemu, also it seems the libvirt group is hardcoded, so we'd still need a "libvirt" group along with an _libvirt group